### DISCUSSION TITLE
Exit code is zero even when the simulation fails

### DISCUSSION THREAD

**Discussion by Pbarbosa92** [firemodels/fds#12511] [General]
Hi guys,
I would like to know if this is the expected behavior. When the simulator fails, the return code is still zero.
![Captura de ecrã 2024-02-21, às 15 00 02](https://github.com/firemodels/fds/assets/157368229/283f4ec5-5278-4c5d-ba28-844f9b189da2)
Am I missing something? Or is this the expected behavior?

Thanks in advance ;)

**Comment by drjfloyd** [comment]
FDS does not have a --help option, it is expecting the name of your input file. See the User's Guide for details on running FDS on Linux.

**Reply by Pbarbosa92** [reply]
I know it doesn't. I wanted fds to fail to see the return value. And the return value was zero.

**Comment by drjfloyd** [comment]
FDS error codes are only written to stderr

**Reply by Pbarbosa92** [reply]
Thanks for the reply, that is what I wanted to know ;)
